OpenRA.Mods.Common/Projectiles/Bullet.cs

```diff
@@ -63,6 +63,10 @@ public class BulletInfo : IProjectileInfo
 		[Desc("Arc in WAngles, two values indicate variable arc.")]
 		public readonly WAngle[] LaunchAngle = { WAngle.Zero };
 
+		[Desc("Altitude where this bullet should explode when reached.",
+			"Negative values allow this bullet to pass cliffs and terrain bumps.")]
+		public readonly WDist ExplodeUnderThisAltitude = WDist.Zero;
```

Comment:
Well if you go less diplomatic...

Implementing this as-is allow one to create a straight-fire projectile without exploding on terrain bumps, something old Bullet could do. Something classic C&C modders like to do but can't - http://ppmforums.com/viewtopic.php?p=186779#186779 explains why.

Say no and I will close this PR and shift this option into an AS duplicate then implement `AirburstAltitude` in another PR because the two are unrelated.